JSDemos/Demos/DataGrid/MasterDetailView/Angular/app/app.component.ts

```diff
@@ -19,34 +19,8 @@ if (!/localhost/.test(document.location.host)) {
 })
 export class AppComponent {
     employees: Employee[];
-    tasks: Task[];
-    tasksDataSourceStorage: any;
     constructor(private service: Service) {
-        this.employees = service.getEmployees();
-        this.tasks = service.getTasks();
-        this.tasksDataSourceStorage = [];
-    }
-
-    completedValue(rowData) {
-        return rowData.Status == "Completed";
-    }
-
-    getTasks(key) {
```

Comment:
I see only one issue in removing this function - now we do not have a way that demonstrates how to avoid the change detection hell in our demos. But, from the code perspective, we do not need to create inline templates, they are awful.